openbb_platform/core/openbb_core/provider/standard_models/equity_historical.py

```diff
@@ -20,10 +20,6 @@ class EquityHistoricalQueryParams(QueryParams):
     """Equity Historical Price Query."""
 
     symbol: str = Field(description=QUERY_DESCRIPTIONS.get("symbol", ""))
-    interval: Optional[str] = Field(
```

Comment:
This changes on the standard models, namely adding/removing arguments should be done in a different PR unless there's a reason to do it on this one - it seems unrelated.

Reply:
This is all related because now these choices are actually in `openapi.json` and `reference.json`, where they were not previously.

Reply:
fair enough